content/components/tooltip.mdx

```diff
@@ -3,42 +3,239 @@ title: Tooltip
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="456"
+      alt="Pointer hovering over a link with no tooltip"
+      src="https://github.com/primer/primitives/assets/18661030/d62b048f-1095-41d8-8bfb-fef204ba50dc"
+    />
+    <Caption>Remove any duplicate tooltip text.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="456"
+      alt="Screenshot of tooltip with GitHub username, @inkblotty, on a static span that says @inkblotty"
+      src="https://github.com/primer/primitives/assets/18661030/78046a69-57f2-4fcd-9f17-fc06576d704f"
+    />
+    <Caption>Don't set tooltips that duplicate the trigger element content.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### 3. Use a modal
+
+Consider using a modal, which is accessible for mobile users and allows you to structure content that may otherwise be crammed into a tooltip.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="456"
+      alt="Screenshot of GitHub Actions page with a modal that conveys workflow file information about a runner"
+      src="https://user-images.githubusercontent.com/40274682/160210527-2319e408-d433-407f-aac2-66eaf9e63246.png"
+    />
+    <Caption>Do use a pattern that is accessible to a larger number of users such as a modal</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="456"
+      alt="Screenshot of GitHub Actions page using a tooltip on a static element to convey long workflow file information"
+      src="https://github.com/primer/primitives/assets/18661030/f1107770-cf3b-4bc1-9f75-a766833c67e0"
+    />
+    <Caption>Don't place long content inside a tooltip</Caption>
+  </Dont>
+</DoDontContainer>
+
+### 4. Use a summary disclosure
+
+Consider using a summary disclosure that is accessible for mobile users and also offers the ability to show or hide content.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="800"
+      alt="Screenshot of GitHub pricing page that uses a summary disclosure to hide and show additional information on each point"
+      src="https://user-images.githubusercontent.com/40274682/160211684-6c102dd6-4039-4c7b-8c52-eeeb71a27db5.png"
+    />
+    <Caption>Do use a pattern that is accessible to a larger number of users such as a summary disclosure</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="800"
+      alt="Screenshot of GitHub pricing page using a tooltip to hide and show long descriptions on each point"
+      src="https://github.com/primer/primitives/assets/18661030/8efbc68b-7fa9-409d-a020-2e5d75337005"
+    />
+    <Caption>Don't use tooltips on non-interactive elements</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Other
+
+If you are unsure which alternative is more suited to your scenario and need help, consult a designer or the GitHub Accessibility team (if you are GitHub staff) for advice.
 
-Most UI cases don't call for tooltips. Learn some [alternative methods to use in place of tooltips](/../guides/accessibility/tooltip-alternatives).
+## Additional resources
```

Comment:
We [introduced a new PVC lint rule](https://github.com/primer/view_components/pull/2086) recently the deprecated Primer CSS tooltipped which we have a ton of debt around that we want to tackle. It looks like the lint rule doc is now on [Primer CSS Tooltipped](https://primer.style/design/guides/development/rails/migration-guides/primer-css-tooltipped).
Could it make sense to link to it here? :)